safe-global/safe-wallet-web
# feat: recovery proposal flow

## What it solves

Resolves #2764

## How this PR fixes it

This adds a new transaction flow for proposing a recovery attempt from a template "Recovery" section of the settings (later to be moved).

## How to test it

Open the "Recovery" section of a Safe and observe a new area that allows for the enabling of recovery. Ensuring that the recovery has been enabled (possible via the same section), a "Propose recovery" button will take you through a new flow according to [the designs](https://www.figma.com/file/pO43pjVwR34Uy1ihMsY3OM/Generic-recovery?type=design&node-id=456-15322&mode=design&t=Lo3nDvw622nW8RA7-0).

## Screenshots

![propose-recovery](https://github.com/safe-global/safe-wallet-web/assets/20442784/eb0939f3-daa2-46a8-945e-de67372d004b)

## Checklist
